charts/fybrik/templates/manager-deployment.yaml

```diff
@@ -80,18 +80,35 @@ spec:
             {{- end }}
           env:
             - name: ENABLE_WEBHOOKS
+            {{- if .Values.clusterScoped }} 
               value: "true"
+            {{- else }}
+              value: "false"
+            {{- end }}
```

Comment:
Can probably use `value: {{ .Values.clusterScoped | quote }}` or something similar.

Reply:
Unless you insist I am inclined to leave as is. Setting it explicitly simplifies the check in the go code.

Reply:
I don't see how the go code is related but ok I don't really mind.